src/tls/cockpit-certificate-helper.in

```diff
@@ -2,51 +2,88 @@
 
 set -eu
```

Comment:
I'm fairly sure we can use `set -f` here as wel, just in case the cert names contain funny characters.

Reply:
Is there anywhere that isn't properly quoted that risks expansion?

I'd prefer to do this locally correct than to set this global flag...

Reply:
No, I didn't see insufficient quoting. It's more like belt-and-suspenders - i.e. we don't need glob expansion, so it could just as well disable it.